### PR TITLE
chore(upgrade): upgrade golang (1.13.4) and serverless (1.57)

### PR DESCRIPTION
**Pull request by catsoap**
some fixes on golang https://github.com/golang/go/issues?q=milestone%3AGo1.13.4  
and a little upgrade for serveless

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ java/Dockerfile
 php/Dockerfile
 python/Dockerfile
 serverless/Dockerfile
+golang/Dockerfile
```
Comment (mremi) [inline .gitignore]: please sort

File: config.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@ dind-aws:
         - docker-compose --version
 
 golang:
-  "1.13":
+  "1.13.4":
```
Comment (mremi) [inline config.yml]: to be reverted

File: .gitignore (hunk 1 above)
Comment (catsoap) [reply .gitignore]: sorry done

File: config.yml (hunk 2 above)
Comment (catsoap) [reply config.yml]: :man_facepalming: oups, done

File: config.yml [hunk 3]
```diff
@@ -310,11 +310,11 @@ scoutsuite:
         - scout --version
 
 serverless:
-  "1.51":
+  "1.57":
```
Comment (mremi) [inline config.yml]: requires to update `.travis.yml` as well (that will fix Travis)

Comment (catsoap) [reply config.yml]: :sweat: thanks, repushed

**Pull request merged**